stubs/Pillow/PIL/ImageDraw.pyi

```diff
@@ -1,21 +1,25 @@
 from collections.abc import Container
-from typing import Any, Tuple, Union, overload
+from typing import Any, Sequence, Tuple, Union, overload
 from typing_extensions import Literal
 
 from .Image import Image
 from .ImageFont import _Font
 
 _Ink = Union[str, Tuple[int, int, int]]
+_XY = Sequence[Union[float, Tuple[float, float]]]
```

Comment:
Can this really be `[(1, 2)]` or something? Or should it be `Union[Sequence[float], Tuple[float, float]]` or just `Sequence[float]`?

Reply:
My original stubs were stricter about this, but I went and relooked at everything before sending the PR, and they have this function `PyPath_Flatten` that gets run on all of the `xy` parameters which seems to allow things as general as this.

https://github.com/python-pillow/Pillow/blob/60d6dc78695db6204ddfdcb45d7795ebb28e02d0/src/path.c#L111

Reply:
The function amounts to "iterate over xy, pulling out each number, or potentially a 2-tuple of doubles".